### DISCUSSION TITLE
Search functionality for Table Component?

### DISCUSSION THREAD

**Discussion by TylerWilson61** [ant-design-blazor/ant-design-blazor#2822] [Ideas]
Is it possible to add search functionality to the Table component? I am thinking of something like [this](https://www.mudblazor.com/components/table#table-with-pagination-and-filtering) search functionality in the MudBlazor table component.

I am currently using ant blazor in production for a metadata explorer project. The underlying data is stored in multidimensional cubes and queried with MDX. I chose ant blazor because I was able to use nested tables easily (useful for visualizing multidimensional objects).

However, there is no way to search for any specific cell in the table. Is this feature possible? Thanks.

**Comment by michaelcsikos** [comment]
Hi @TylerWilson61, did you make any progress on this? The MudBlazor search looks great.

**Comment by ElderJames** [comment]
I think it just need to add a `Search` component on the top of Table or the `Title` Template of table. And mix the search string into query parameters.

I will update the demo.
